packages/setup/src/index.ts

```diff
@@ -165,65 +105,39 @@ const run = async (): Promise<void> => {
   const rootEnv = {
     NODE_ENV: answers.NODE_ENV,
     HOST: answers.HOST,
-    API_PORT: answers.API_PORT,
+    API_PORT: process.env.API_PORT,
     SERVER_URL:
       answers.NODE_ENV === 'production'
         ? `https://${answers.HOST as string}`
-        : `http://${answers.HOST as string}:${answers.API_PORT as string}`,
+        : `http://${answers.HOST as string}:${process.env.API_PORT as string}`,
     FRONTEND_URL:
       answers.NODE_ENV === 'production'
         ? `https://${answers.HOST as string}`
-        : `http://${answers.HOST as string}:${answers.PORT as string}`,
+        : `http://${answers.HOST as string}:${process.env.PORT as string}`,
     MONGO_HOST: answers.NODE_ENV === 'production' ? 'mongodb' : 'localhost',
-    MONGO_INITDB_ROOT_USERNAME: answers.MONGO_INITDB_ROOT_USERNAME,
-    MONGO_INITDB_ROOT_PASSWORD: answers.MONGO_INITDB_ROOT_PASSWORD,
-    MONGO_USERNAME: answers.MONGO_USERNAME,
-    MONGO_PASSWORD: answers.MONGO_PASSWORD,
-  };
-  await setupAndWriteEnv(
-    '/usr/praise/.env.template',
-    '/usr/praise/.env',
-    rootEnv
-  );
-
-  const apiEnv = {
+    MONGO_INITDB_ROOT_USERNAME: randomString(12),
```

Comment:
Don't generate random usernames for root username and username. Use the ones specified in `.env.template`